webapp/locales/en.json

```diff
@@ -444,8 +444,9 @@
     },
     "comment": {
       "submit": "Comment",
-      "submitted": "Comment Submitted",
-      "updated": "Changes Saved"
+      "submitted": "Comment submitted!",
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-09T08:07:36Z" title="Thursday, January 9th 2020, 9:07:36 am +01:00">Jan 9, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -423,7 +423,8 @@
     "comment": {
       "submit": "Comment",
       "submitted": "Comment Submitted",
-      "updated": "Changes Saved"
+      "updated": "Changes Saved",
+      "answered": "answered"
```


I would name this `"answer": "Answer"`.
It should be the present form, because you will do it right now – answer it.
Answered is the past form.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-10T07:20:15Z" title="Friday, January 10th 2020, 8:20:15 am +01:00">Jan 10, 2020</time>_

---


ok